src/modality-specific-files/task-events.md

```diff
@@ -309,3 +309,95 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
     "VisionCorrection": "lenses"
 }
 ```
+
+### Continuously-sampled, stimulus-related signals
+
+!!! example "Example datasets"
+
+      At the time of writing, there are no
+      [example datasets](https://bids-standard.github.io/bids-examples/#dataset-index)
+      showcasing stimulus-related signals.
+      However, the following datasets shared on [OpenNeuro](https://openneuro.org)
+      include these type of signals and may be used (with caution, because they are not
+      "official" examples) as a reference when curating a new dataset:
+
+      -   [*Trial timing for multivariate pattern
+          analysis*](https://openneuro.org/datasets/ds000238/versions/00002)
+
+Signals related to stimuli (such as parameters of a film or audio stimuli) that are
+evenly recorded at a constant sampling frequency SHOULD be stored as follows.
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    <datatype>/
+        <matches>[_recording-<label>]_stim.tsv.gz
+        <matches>[_recording-<label>]_stim.json
+```
+
+For the template directory name, `<datatype>` can correspond to any data
+recording modality.
+
+In the template filenames, the `<matches>` part corresponds to task filename
+before the suffix.
+For example for the file `sub-control01_task-nback_run-1_bold.nii.gz`,
+`<matches>` would correspond to `sub-control01_task-nback_run-1`.
+
+!!! warning "Caution"
+
+    `<matches>[_recording-<label>]_stim.tsv.gz` files MUST NOT include a header line,
+    as established by the [common-principles](../common-principles.md#compressed-tabular-files).
+    As a result, when supplying a `<matches>[_recording-<label>]_stim.tsv.gz` file,
+    an accompanying `<matches>[_recording-<label>]_stim.json` MUST be present to indicate
+    the column names.
+
+The [`recording-<label>`](../appendices/entities.md#recording)
+entity MAY be used to distinguish between several recording files,
+as prescribed by the specifications for
+[physiological recordings](physiological-recordings.md).
```

Comment:
This is transferred directly from the original location in physio, but I'm not sure this applies anymore. I'm in favor of removing specific mentions to `[_recording-<label>]`.

```suggestion
The [`recording-<label>`](../appendices/entities.md#recording)
entity MAY be used to distinguish between several recording files,
as prescribed by the specifications for
[physiological recordings](physiological-recordings.md).
```

Reply:
I am reading this as a "soft" removing of the recording label for stim files. Let me know if this is not the correct interpretation of your comment.

I agree that this recording label now reads strange out of the physio context.

However we've always had it and it's been part of the schema for stim files for a long time now, so I think it would be breaking backward compatibility to remove it.

Reply:
Yes, a soft removal of the recording label.